Enterprise-CMCS/cmcs-eregulations
# EREGCSC-2450 -- Remove "didn't find what you were looking for?" from SearchInput

Resolves [EREGCSC-2450](https://jiraent.cms.gov/browse/EREGCSC-2450)

**Description**

The Policy Repository and Combined Search page use the generic `SearchInput` Vue component to search for content.  This `SearchInput` component has built-in functionality that allow it to display suggestions underneath the input element. One of the suggestions ist can offer is to surround a multi-word query with quotation marks to return more precise results.

Unfortunately, the multi-word suggestion is not working properly in the Policy Repository, and it is also confusing users on the Combined Search Page.  As such, the multi-word suggestion will now be hidden wherever the `SearchInput` component is used.  It will also be removed from the bespoke search implementation on the Resources page.

It should be noted that Synonyms will still appear beneath the search input on the Combined Search and Resources pages.

**This pull request changes:**

- Adds boolean `showSuggestions` prop to `SearchInput` that defaults to `false`.
     - This retains the multi word query suggestion functionality, but makes it an "opt in" feature
     - To opt in to this feature, pass in `show-suggestions=true` when using the `SearchInput` component.
     - Currently, no implementation of the `SearchInput` component opts in to this feature.

**Steps to manually verify this change:**

1. Visit the [policy repository on the experimental deployment](https://p6p4j0xig0.execute-api.us-east-1.amazonaws.com/dev1147/policy-repository/)
2. Type a multi-word query into the search input at the top of the left sidebar and execute search
3. Notice that there are no "didn't find what you were looking for?" suggestions being displayed beneath the search input
4. Compare to `prod` policy repository, which will show a multi-word suggestion beneath the search input
5. Visit [Search](https://p6p4j0xig0.execute-api.us-east-1.amazonaws.com/dev1147/search/) and [Resources](https://p6p4j0xig0.execute-api.us-east-1.amazonaws.com/dev1147/resources/) page on the experimental deployment 
6. Type a multi-word query, execute search, and ensure that there are no "didn't find what you were looking for?" suggestions being displayed beneath the search input
7. Search for "FMAP" to ensure that synonym suggestions are still being shown (Combined Search and Resources pages only)
